README.md

```diff
@@ -10,14 +10,16 @@ Manages OS users from databags.
 
 ## Scope
 
-This cookbook is concerned with the management of OS users and groups from databags. It also manages the distribution of ssh keys to a user's home directory.
+This cookbook is concerned with the management of OS users and groups. It also manages the distribution of ssh keys to a user's home directory.
```

Comment:
I think this might work better.
```suggestion
This cookbook is concerned with the management of OS users and groups (optionally from databags). It also manages the distribution of ssh keys to a user's home directory.
```